docs/podman-restart.1.md

```diff
@@ -0,0 +1,45 @@
+% podman(1) podman-restart - Restart a container
+% Matt Heon
+# podman-restart "1" "March 2017" "podman"
+
+## NAME
+podman restart - Restart a container
+
+## SYNOPSIS
+**podman attach [OPTIONS] CONTAINER [CONTAINER...]**
+
+## DESCRIPTION
+The restart command allows containers to be restarted using their ID or name.
+Containers will be stopped if they are running and then restarted.
```

Comment:
This implies if a container is stopped, restart will just start it? If so, probably worth saying.